chapters/1.md

```diff
@@ -11,33 +11,39 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
 
-Thankfully, an emerging new blockchain technology is capable of mitigating all of the above risks by creating a distributed ledger that all parties can equally use, view, and verify. This remarkable capability for strangers to agree on a shared document, which is called decentralized consensus, has been revolutionized in the last decade.
+The overarching challenge was to create a completely verifiable monetary system without needing to trust a central third party. By decentralizing power over issuance and transaction processing, it was theorized that the economic participants could collectively keep each other honest and resist attempts at manipulation.
 
-It is easy to be confused about the terminology at first, especially since most people are simultaneously introduced to several jargony concepts. You can think about "blockchains" as a technology that allows networks to establish "decentralized consensus" agreements. By enabling strangers to safely share a ledger, it becomes possible to build "cryptocurrencies" that function as digital cash. There are a multitude of regular currencies (euros, dollars, yen, etc); analogously, various teams have built many different cryptocurrencies (Monero, Ethereum, Bitcoin, etc).
+One significant obstacle in creating such a system was that there was no known way to prevent people from spending the same money twice without having a central party to validate each transaction.  Given that some network participants may try to lie, cheat, pretend to be multiple people, etc, how do you maintain a single version of the ledger to prevent double spending?  In computing, this is illustrated in a thought experiment referred to as the “Byzantine Generals problem” meant to demonstrate the pitfalls and design challenges of attempting to coordinate an action by communicating over an unreliable network. In the experiment, two generals are only able to communicate with one another by sending a messenger through enemy territory. The experiment asks how they might reach an agreement on the time to launch an attack, while knowing that any messenger they send could be captured.
+
+Thankfully, in 2009 with the launch of Bitcoin, Satoshi Nakamoto (a pseudonym) demonstrated a suitable combination of technologies, game theory and economics which mitigated the above issues.  This gave the world an open, verifiable, trustless currency which anyone could use, but no single entity controlled.  In the first 4 years of its existence, a number of shortcomings were identified in Bitcoin's original design and implementation. In 2013 Nicolas van Saberhagen (also a pseudonym) published the CryptoNote 2.0 whitepaper with the goal of addressing these shortcomings and provide a superior digital cash payment system.  Six months later, Monero was launched as an implementation of the CryptoNote protocol.
```

Comment:
I would like to remove these paragraphs: the content is going to be redundant (look for 1.2.1 section).

Reply:
Done.